qiskit-ibm-transpiler-mcp-server/pyproject.toml

```diff
@@ -0,0 +1,84 @@
+[project]
+name = "qiskit-ibm-transpiler-mcp-server"
+version = "0.1.0"
+description = "MCP server for AI Transpiler IBM Quantum services through local and Qiskit IBM Runtime executions"
+readme = "README.md"
+requires-python = ">=3.10,<3.14"
+
+dependencies = [
+    "fastmcp>=2.12.4",
+    "qiskit-ibm-transpiler>=0.15.0",
+]
+[project.optional-dependencies]
+dev = [
+    "ruff>=0.11.13",
+    "mypy>=1.15.0"
+]
+
+test = [
+    "pytest>=8.4.2",
+    "pytest-asyncio>=1.2.0",
+    "pytest-mock>=3.15.1",
+    "pytest-cov>=4.1.0",
+    "bandit[toml]>=1.9.2"
+]
```

Comment:
This defines the same as lines 66-78 in this file https://github.com/Qiskit/mcp-servers/pull/27/files#diff-73e378a7edae0025c582ebf015449376c103d0f921d30f04e7e92b3e2fa4d661R66-R78, so it's redundant.

Use only one approach (preferably `[project.optional-dependencies]` for compatibility).